docs/evm/cadence/batched-evm-transactions.md

```diff
@@ -0,0 +1,641 @@
+
+Copy your transaction ID and go to the Flowscan Testnet Cadence block explorer: [Flowscan
+Cadence](https://testnet.flowscan.io/).
+
+Pasting your transaction ID into the search bar will show you the transaction details, including the Cadence script,
+execution status, and event logs. Click on the "EVM" tab to view the EVM transactions batched in the Cadence
+transaction.
+
+![Embedded EVM transactions on Flowscan](./evm-embed-flowscan.png)
+
+Clicking on the transactions will open up the EVM transaction in Flowscan's EVM block explorer. If you view the EVM
+transactions in order, you'll notice that they aggregate the same actions we took manually in the MetaMask section, but
+this time in a single Cadence transaction!
+
+:::tip
+
+At this point you may realize there are two block explorers on Flow - one for Cadence activity and another for EVM
+activity. This is unique to Flow and is a consequence of the fact that Cadence & EVM are separate runtimes, with EVM
+effectively emulated within Cadence. This orientation - that of EVM running within Cadence means that the Cadence-side
+explorer has visibility to EVM transactions embedded within a Cadence transaction.
+
+Practically, this means that any transactions run using a Flow account can be viewed on the Cadence explorer while any
+transactions run using an EVM account can be viewed on the EVM explorer.
+
+:::
+
+## Breaking it Down
+
+Now that we can relate to the pain of manually executing these transactions and we've seen the magic you can work with
+Cadence, let's understand what's going on under the hood.
+
+To recap, our Cadence transaction does the following, reverting if any step fails:
+
+1. Wraps FLOW as WFLOW
+2. Approves the `MaybeMintERC721` contract to move WFLOW
+3. Attempts to mint a `MaybeMintERC721` token
+
+But how does our Flow account interact with EVM from the Cadence runtime? As you'll recall from the [Interacting with
+COA](./interacting-with-coa.md) guide, we use a Cadence-owned account (COA) to interact with EVM contracts from
+Cadence.
+
+A COA is a [resource](https://cadence-lang.org/docs/solidity-to-cadence#resources) providing an interface through which
+Cadence can interact with the EVM runtime. This is importantly ***in addition*** to the traditional routes you'd
+normally access normal EVMs - e.g. via the JSON-RPC API. And with this interface, we can take advantage of all of the
+benefits of Cadence - namely here scripted transactions and conditional execution.
+
+So in addition to the above steps, our transaction first configures a COA in the signer's account if one doesn't already
+exist. It then funds the COA with enough FLOW to cover the mint cost, sourcing funds from the signing Flow account's
+Cadence Vault. Finally, it wraps FLOW as WFLOW, approves the ERC721 contract to move the mint amount, and attempts to
+mint the ERC721 token.
+
+Let's see what each step looks like in the transaction code.
+
+### COA Configuration
```

Comment:
Would it be possible to guide the learner through **building** this script along the way, testing it one step at a time?

Reply:
Good suggestion. I'll work on expanding these sections into their own transactions with follow up scripts, I think you're right and that would enhance learning.

Reply:
Added in [4375b69](https://github.com/onflow/docs/pull/983/commits/4375b69cfeb6214c32c33a1349d9f9e85f537c60)